_ideas/heatmap/intermediate/2-cell-hover-count.md

```diff
@@ -0,0 +1,12 @@
+## Heatmap - Intermediate
+
+## Task
+When a cell in the heatmap is hovered, display a count of the number of posts.
+
+## Overview
+To display a count on hover, you'll need to use D3's  to the heatmap cells. You may find [this guide](https://medium.com/@kj_schmidt/show-data-on-mouse-over-with-d3-js-3bf598ff8fc2) helpful.
```

Comment:
Typo? 
> you'll need to use D3's  to the heatmap cells

Guide is good resource though

Reply:
Resource https://medium.com/@kj_schmidt/show-data-on-mouse-over-with-d3-js-3bf598ff8fc2 uses D3.js version 4.6.0. Is this a problem?

Reply:
I think this is okay, because the API for the mouseover hasn't changed!